lib/modules/apostrophe-areas/lib/api.js

```diff
@@ -232,7 +232,12 @@ module.exports = function(self, options) {
           type: 'area',
           items: items
         });
-        return self.apos.docs.update(req, doc, callback);
+        
+        if(self.apos.docs.getManager(doc.type) != undefined && self.apos.docs.getManager(doc.type).update != undefined){
```

Comment:
Thanks!

A few quick things:

* Avoid redundant code, please set a manager variable rather than fetching it twice.
* A simple if without != undefined would be more appropriate.
* Also please follow the coding style, space after "if", space before "callback".

Reply:
You can type `npx eslint .` to see what coding style issues are flagged automatically. `npx eslint . --fix` can fix some.